File: app/Models/Expense.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Expense extends Model
{
    use HasFactory;

    protected $fillable = ['budget_id', 'category', 'amount'];

    public function budget()
    {
        return $this->belongsTo(Budget::class);
    }

    public function user()
    {
        return $this->belongsTo(User::class);
    }
}

File: MAUIAPP/BudgetPlaner/Models/Expense.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BudgetPlaner.Models;

public class Expense
{
    public int Id { get; set; }
    public int Budget_Id { get; set; }
    public string Category { get; set; }
    public double Amount { get; set; }
}
